src/components/study-manager.js

```diff
@@ -133,43 +133,55 @@ const StudyCard = ({study, onClick}) => {
         />
     ));
 
+    // ########################################################################
```

Comment:
We don't have things like in our source code.. Not sure about adding them. @geofjamg ?

Reply:
Personnally I hate this kind of ascii art separator. It doesn't clarify anything just:

// action menu

is enough.

Reply:
I agree it was unconventional and ugly here, but could we consider documenting the Front by using a standard like this?
- https://react-styleguidist.js.org/docs/documenting.html

Personally I regret the lack of comment, even though I haven't put much into it myself so far. It's superfluous for some, but more readable for others. This will be essential at least for the common-ui repo, so I could start here.

Reply:
Everybody is welcome to add as much comment as possible as far as it is done in a coventional and normalized way.